doc/apps/dash.md

```diff
@@ -30,6 +30,18 @@ app = Dash(__name__)
 server = app.server
 ```
 
+```{note}
+- `server = app.server` - Required to start a production grade server. Without it, deployment will fail as the infrastructure cannot find the server to start.
+- `app = Dash(__name__)` - The deployment may require modification of the application's basic routing. If deployment fails, check the following log:
+
+  _ERROR: We were not able to modify your Routing automatically_
+
+  _INSTRUCTION: Please add `url_base_pathname="/application/"` to `app = Dash(__name__, ...)`_
```

Comment:
does this happen because of the marketing banner?

Reply:
Yes, for this comment: https://github.com/ploomber/cloud-backend/pull/1626#discussion_r1861149148

Bryan recommended to add a notice
- But the replace should always work, since the search is done in every python file of the project. (I handle with multiple parameter, if the `url_base_pathname` is already there or not, and more..)
- So we could not mention it in the Doc. And if ever if fails (it should not), a is log emitted.

Reply:
(So this PR is optional)